themes/default/content/blog/kubernetes-fundamentals-part-two/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Kubernetes Fundamentals Part Two"
+date: 2021-09-02T12:54:53-05:00
+draft: false
+meta_image: k8s-fundamentals.png
+meta_desc: "Kubernetes is everywhere now, but it’s largely been the domain of people who work on the Ops side of things. What about devs, though?"
+authors:
+    - kat-cosgrove
+tags:
+    - kubernetes
+    - google-cloud
+---
+Kubernetes is everywhere now, but it’s primarily been the domain of people working on the Ops side of infrastructure. What about devs, though? You benefit from knowing what Kubernetes is and how to use it, too -- otherwise, we’re still putting teams in silos. In this blog, we're going to build off part one by learning about managed kubernetes services: what they are, when they're useful, and how you can try deploying to one yourself, starting with Google's GKE.
```

Comment:
```suggestion
Kubernetes is everywhere now, but it’s primarily been the domain of people working on the ops side of infrastructure. What about devs, though? You benefit from knowing what Kubernetes is and how to use it, too&mdash;otherwise, we’re still putting teams in silos. In this blog, we're going to build off part one by learning about managed Kubernetes services: what they are, when they're useful, and how you can try deploying to one yourself, starting with Google's Kubernetes Engine (GKE).
```